examples/inlinekeyboard.py

```diff
@@ -36,20 +36,26 @@ def help(bot, update):
 
 
 def error(bot, update, error):
-    logging.warning('Update "%s" caused error "%s"' % (update, error))
+    logger.warn('Update "%s" caused error "%s"', update, error)
```

Comment:
Can you change this back to `logger.warning`? `Logger.warn` is deprecated.
https://docs.python.org/3/library/logging.html#logging.Logger.warning